packages/frontend/src/components/dialogs/CreateChat/index.tsx

```diff
@@ -258,6 +258,17 @@ function CreateChatMain(props: CreateChatMainProps) {
 
   const fixedSizeListOuterRef = useRef<HTMLElement>(null)
 
+  const onKeyDown = (ev: React.KeyboardEvent) => {
+    if (ev.code === 'ArrowDown') {
+      ;(
+        fixedSizeListOuterRef.current?.firstElementChild?.firstElementChild
+          ?.firstElementChild?.firstElementChild as HTMLElement
```

Comment:
Not sure if that is a reliable path. Maybe @WofWca have a look at it, since he introduced the fixedSizeListOuterRef

Reply:
selects the first visible/loaded item, we could probably scroll up before to make this select the first item?

Reply:
Yeah, this doesn't look too good. `querySelector('button')` or alike would be better I'd say, similar to here

https://github.com/deltachat/deltachat-desktop/blob/bbd353d68b83ea537e6f6735a5abb6e54c4a6b0c/packages/frontend/src/components/chat/ChatList.tsx#L294-L303